debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -410,6 +410,14 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                            .withDependents(DATABASE_WHITELIST_NAME)
                                                            .withDescription("Flag specifying whether built-in tables should be ignored.");
 
+    public static final Field JDBC_DRIVER = Field.create("database.jdbc.driver")
+            .withDisplayName("Jdbc Driver Class Name")
+            .withType(Type.STRING)
+            .withWidth(Width.MEDIUM)
+            .withDefault("com.mysql.jdbc.Driver")
```

Comment:
Would it be better to define the default value as `com..mysq.jdbc.Driver.class.getName()` rather than a literal string?